### ISSUE TITLE
Set Up CSS

### ISSUE THREAD

**Issue by BMArsenault** [BMArsenault/proj-01-HikeSpot#3]
- [x] Find colors to use for background/text/buttons/etc.   Possibly a green, brown and maybe orange to go with hiking colors...
- [x] Move div boxes to our liking on the page.  Search on top left (?) - weather underneath search (?) - map to the right (?)

**Comment by BMArsenault** [comment]
Looked at the color palette Jonathan posted in resources.  Here are some options I liked but I'm not stuck on them.  There are 4 colors in each palette but can obviously just use 2 or 3 of the colors.

https://colorhunt.co/palette/125c133e7c17f4a442e8e1d9

https://colorhunt.co/palette/ffce45d4ac2bb05e277e370c

https://colorhunt.co/palette/22577a38a3a557cc9980ed99

**Comment by BMArsenault** [comment]
Hey!  Take a look at the uploaded html and css in the ba/feature/js branch.  @K2S00 When I added the bootstrap styling to the html, the whole page went bonkers so I made a few changes and added background image.  My only issue I had with this is for some reason the "Trail Map" and "Weather" had a text box border that I could not get rid of so I just put "display: none;" to hide it right now.  As stated earlier, I'm not stuck on the color scheme so we can switch that around.

Also, not sure how this works with you guys getting updates so I'll also tag Hilary and Christian to make sure it pops up... somewhere.  @hpurring @Christian55diaz
